com.unity.render-pipelines.universal/Runtime/LightCookieManager.cs

```diff
@@ -11,18 +11,22 @@ static class ShaderProperty
         {
             public static readonly int mainLightTexture = Shader.PropertyToID("_MainLightCookieTexture");
             public static readonly int mainLightWorldToLight = Shader.PropertyToID("_MainLightWorldToLight");
-            public static readonly int mainLightCookieFormat = Shader.PropertyToID("_MainLightCookieFormat");
+            public static readonly int mainLightCookieTextureFormat = Shader.PropertyToID("_MainLightCookieTextureFormat");
 
             public static readonly int additionalLightsCookieAtlasTexture = Shader.PropertyToID("_AdditionalLightsCookieAtlasTexture");
-            public static readonly int additionalLightsCookieAtlasFormat = Shader.PropertyToID("_AdditionalLightsCookieAtlasFormat");
+            public static readonly int additionalLightsCookieAtlasTextureFormat = Shader.PropertyToID("_AdditionalLightsCookieAtlasTextureFormat");
 
-            public static readonly int additionalLightsCookieAtlasUVRectBuffer = Shader.PropertyToID("_AdditionalLightsCookieAtlasUVRectBuffer");
-            public static readonly int additionalLightsWorldToLightBuffer = Shader.PropertyToID("_AdditionalLightsWorldToLightBuffer"); // TODO: really a light property
-            public static readonly int additionalLightsLightTypeBuffer = Shader.PropertyToID("_AdditionalLightsLightTypeBuffer"); // TODO: really a light property
+            public static readonly int additionalLightsCookieEnableBits = Shader.PropertyToID("_AdditionalLightsCookieEnableBits");
 
+            public static readonly int additionalLightsCookieAtlasUVRectBuffer = Shader.PropertyToID("_AdditionalLightsCookieAtlasUVRectBuffer");
             public static readonly int additionalLightsCookieAtlasUVRects = Shader.PropertyToID("_AdditionalLightsCookieAtlasUVRects");
-            public static readonly int additionalLightsWorldToLights = Shader.PropertyToID("_AdditionalLightsWorldToLights"); // TODO: really a light property
-            public static readonly int additionalLightsLightTypes = Shader.PropertyToID("_AdditionalLightsLightTypes"); // TODO: really a light property
+
+            // TODO: these should be generic light property
```

Comment:
You know me :)
Can we kill the TODO and make a Jira card about this task?

Reply:
I'll do a task for it.